docs/UserGuide.md

```diff
@@ -154,6 +159,23 @@ Exits the program.
 
 Format: `exit`
 
+### Flipping a flashcard : `flip`
+
+* Flips the flashcard at the specified `INDEX` to either show or hide the translation to the displayed word or phrase.
+* Similar to delete, the index refers to the index number shown in the displayed flashcard list.
+* Similar to delete, the index **must be a positive integer** 1, 2, 3, …​
```

Comment:
Can we follow the format of the rest of the documentation? For example

```
### Deleting a person : `delete`
Deletes the specified person from the address book.
Format: `delete INDEX`
* Deletes the person at the specified `INDEX`.
* The index refers to the index number shown in the displayed person list.
* The index **must be a positive integer** 1, 2, 3, …​
```

I have deployed this commit `6de3fa1` here for reference: https://a8d12ad0.tp-b32.pages.dev/